package.json

```diff
@@ -58,8 +58,8 @@
   },
   "devDependencies": {
     "@ant-design/icons": "4.7.0",
-    "@commitlint/cli": "17.1.2",
-    "@commitlint/config-conventional": "17.1.0",
+    "@commitlint/cli": "17.2.0",
```

Comment:
The `package.json` and `yarn.lock` should match what's in the `master` branch unless you are installing a new package.